tests/ansible.cfg

```diff
@@ -8,7 +8,8 @@ host_key_checking=False
 gathering = smart
 fact_caching = jsonfile
 fact_caching_connection = /tmp
-stdout_callback = skippy
+stdout_callback = default
+display_skipped_hosts = false
```

Comment:
Hi @VannTen 

Do it better to change to `display_skipped_hosts = no` like: https://github.com/kubernetes-sigs/kubespray/blob/master/ansible.cfg#L15 , https://docs.ansible.com/ansible/latest/collections/ansible/posix/skippy_callback.html

It can make the code more readable. :-)


Thanks

Reply:
Done